GenericTest.py

```diff
@@ -70,25 +70,26 @@ def __init__(self, apis, omit_paths=None):
         test = Test("Test initialisation")
 
         for api_name, api_data in self.apis.items():
-            repo = git.Repo(api_data["spec_path"])
+            if "spec_path" in api_data:
+                repo = git.Repo(api_data["spec_path"])
```

Comment:
Suggestion: to avoid yet more lines approaching the 120 character linting limit, would these checks be candidates for writing as
```python
if not ...:
   continue
```
?

Reply:
Good idea